module/VuFind/src/VuFind/Db/Table/Tags.php

```diff
@@ -587,11 +569,12 @@ protected function mergeTags($target, $source)
             return;
         }
         $table = $this->getDbTable('ResourceTags');
+        $tagService = $this->getDbService(\VuFind\Db\Service\TagService::class);
```

Comment:
The Laminas ResourceTags table object is still being used in the line above. Did you leave that intentionally? If so, I have no objection to merging the current work first, and then doing a second PR to get rid of more ResourceTags references... but just mentioning in case it was somehow missed.

Reply:
Sorry, somehow I missed it, but we can move that along with Tags as it has dependency with thin Tags. Please merge the current work.